langchain4j-vertex-ai/src/main/java/dev/langchain4j/model/vertexai/VertexAiEmbeddingModel.java

```diff
@@ -113,6 +144,83 @@ public Response<List<Embedding>> embedAll(List<TextSegment> segments) {
         }
     }
 
+    /**
+     * Calculates the number of tokens for each segment in the input list.
+     *
+     * @param segments a list of TextSegments
+     *
+     * @return a list of tokens counts for each segment
+     */
+    public List<Integer> getTokensCounts(List<TextSegment> segments) {
```

Comment:
minor: is this method `public` on purpose?

Reply:
Yes, it's on purpose, as it's a handy method for people who want to control the size of their prompts.